.scalafmt.conf

```diff
@@ -0,0 +1,44 @@
+version = 2.7.1
```

Comment:
@tszyrowski-MT this is the default auto formatting config we're using on another platui scala project so I copied it in as well, if we can get metals to automatically run the auto formatter on save tomorrow that will be useful 👍